Source/WebGPU/WebGPU/Buffer.h

```diff
@@ -109,7 +109,7 @@ class Buffer : public WGPUBufferImpl, public RefCounted<Buffer>, public CanMakeW
     void indirectIndexedBufferRecomputed(MTLIndexType, NSUInteger indexBufferOffsetInBytes, uint64_t indirectOffset, uint32_t minVertexCount, uint32_t minInstanceCount);
     void indirectBufferInvalidated();
 #if ENABLE(WEBGPU_SWIFT)
-    void copy(const std::span<const uint8_t>, const size_t offset);
+    void copy(const std::span<const uint8_t>, const size_t offset) NS_REFINED_FOR_SWIFT;
```

Comment:
Should we make a DECLARE_SWIFT_CXX_THUNK to do this declaration?

Reply:
DECLARE_SWIFT_CXX_THUNK can do nothing if !ENABLE(WEBGPU_SWIFT) and also add NS_REFINED_FOR_SWIFT at the end.

Reply:
While we're here, is this a copyTo or a copyFrom? Let's update the name.

Reply:
Since the C++ declaration is the interface, I think it makes sense to write it out. But I added a `HAS_SWIFTCXX_THUNK` which expands to NS_REFINED_FOR_SWIFT, to make our intent clearer. That's kind of like the `NS_SWIFT_NAME` attribute that Swift-ObjC interop uses.